3p/integration.js

```diff
@@ -579,6 +580,8 @@ register('zergnet', zergnet);
 register('zucks', zucks);
 register('speakol', speakol);
 
+register('vidoomy',vidoomy);
```

Comment:
nit: space between arguments

Reply:
Added space, and also placed on alphabetically order